arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -361,7 +397,7 @@
         "x-ms-pageable": {
           "nextLinkName": null
         },
-        "operationId": "Profiles_ListAll",
+        "operationId": "Profiles_ListBySubscription",
```

Comment:
Since this is a breaking change, could I get reviewer confirmation that ListBySubscription this is the recommended naming pattern here? Do note that in the generated code (at least in .NET) the method does not actually take a SubscriptionId parameter because that's already built into the client.